src/ast/mod.rs

```diff
@@ -437,19 +437,22 @@ pub enum Expr {
     Cast {
         expr: Box<Expr>,
         data_type: DataType,
+        format: Option<Value>,
```

Comment:
can you please update the documentation here to explain what `format` is (aka what SQL it comes from)?

Reply:
Changed